### PR TITLE
Update Supported Networks

### PR DESCRIPTION
**Pull request by benface**
CleanShot 2023-02-24 at 16 14 23@2x

### CODE REVIEW

**Review by MichaelMacaulay — CHANGES_REQUESTED** [review]
Thank you for kicking this off! 

Could you also add base to the [supported networks](https://thegraph.com/docs/en/developing/supported-networks/) page

**Comment by benface** [comment]
@MichaelMacaulay – Didn't I do that [here](https://github.com/graphprotocol/docs/pull/293/files#diff-66e8c4b758e00618d7a2e7fc40eb19518967df1320e24e0fb679e096606dab11R48)? 🤔

**Comment by MichaelMacaulay** [comment]
@benface you're right - my bad! Been a long week 😅

**Review by MichaelMacaulay — APPROVED** [review]

**Comment by benface** [comment]
@MichaelMacaulay – Haha no worries! I just updated this PR to also move Celo, Avalanche, and Arbitrum One from the "Hosted Service" section to the "The Graph Network & Hosted Service" section. Any chance you could review again? 🙏

File: pages/en/developing/supported-networks.mdx [hunk 1]
```diff
@@ -8,6 +8,11 @@ The following networks are supported on The Graph's Decentralized Network:
 
 - `mainnet` (Ethereum)
 - `gnosis`\*
+- `celo`\*
+- `celo-alfajores`\*
+- `avalanche`\*
+- `arbitrum-one`\*
+- `arbitrum-goerli`\*
```
Comment (benface) [inline pages/en/developing/supported-networks.mdx:15]: @MichaelMacaulay – I'm not 100% sure all of these are supported on The Graph Network, do you know?

Comment (cmwhited) [reply pages/en/developing/supported-networks.mdx:15]: They are supported, @benface. Well, will be as of Monday Feb 27

**Review by cmwhited — APPROVED** [review]

File: pages/en/developing/supported-networks.mdx (hunk 1 above)
Comment (benface) [reply pages/en/developing/supported-networks.mdx:15]: Thank you @cmwhited!

**Pull request merged**